patches/tModLoader/Terraria/release_extras/LaunchUtils/ScriptCaller.sh

```diff
@@ -54,31 +57,31 @@ echo "This may take a few moments."
 source ./DotNetVersion.sh
 
 # Attempt to fix first time Crash To Desktop due to dotnet install failure
-if [[ ! -f "$LaunchLogs/client.log" && ! -f "$LaunchLogs/server.log" ]]; then
+if [[ ! -f "$LaunchLogs/client.log" && ! -f "$LaunchLogs/server.log" && ! "$is_first_run" ]]; then
```

Comment:
Short-circuit FS interactions.
```suggestion
if [[ ! "$is_first_run" && ! -f "$LaunchLogs/client.log" && ! -f "$LaunchLogs/server.log" ]]; then
```